modules/sdk/gradle-plugins-defaults/CHANGELOG.markdown

```diff
@@ -419,6 +419,12 @@ the `com.liferay.app.tlddoc.builder` plugin.
 - [LRDOCS-3023]: Update the [Liferay Gradle Plugins] dependency to version
 3.0.4.
 
+## 1.2.49 - 2016-10-09
+
+### Added
+- [LRDOCS-2594]: apply Liferay stylesheet file, if found, to the API
+documentation generated by the `appJavadoc` and `javadoc` tasks.
```

Comment:
It's this one: https://github.com/liferay/liferay-portal/blob/master/tools/styles/javadoc.css. Yeah, you're right! Maybe we can add a simple link to [Liferay stylesheet](https://github.com/liferay/liferay-portal/blob/master/tools/styles/javadoc.css).